airbyte-webapp/src/components/connection/ConnectionForm/ConnectionConfigurationFormPreview.tsx

```diff
@@ -0,0 +1,104 @@
+  const { getFieldMeta } = useFormikContext<FormikConnectionFormValues>();
+  const allowAutoDetectSchema = useFeature(FeatureItem.AllowAutoDetectSchema);
+
+  const scheduleType = getFieldMeta<ConnectionScheduleType>("scheduleType").value;
+  const scheduleData = getFieldMeta<ConnectionScheduleData>("scheduleData").value;
+  const nonBreakingChangesPreference = getFieldMeta<NonBreakingChangesPreference>("nonBreakingChangesPreference").value;
+  const destNamespaceDef = getFieldMeta("namespaceDefinition").value as NamespaceDefinitionType;
+  const destNamespaceFormat = getFieldMeta("namespaceFormat").value;
+  const destPrefix = getFieldMeta("prefix").value;
+
+  const frequency = (
+    <div>
+      <Text size="xs" color="grey">
+        <FormattedMessage id="form.frequency" />:
+      </Text>
+      <Text size="md" color="grey">
+        {scheduleType === ConnectionScheduleType.manual && <FormattedMessage id="frequency.manual" />}
+        {scheduleType === ConnectionScheduleType.cron && (
+          <>
+            <FormattedMessage id="frequency.cron" /> - {scheduleData?.cron?.cronExpression}{" "}
+            {scheduleData?.cron?.cronTimeZone}
+          </>
+        )}
+        {scheduleType === ConnectionScheduleType.basic && (
+          <FormattedMessage
+            id={`form.every.${scheduleData?.basicSchedule?.timeUnit}`}
+            values={{ value: scheduleData?.basicSchedule?.units }}
+          />
+        )}
+      </Text>
+    </div>
+  );
+
+  const destinationNamespace = (
+    <div>
+      <Text size="xs" color="grey">
+        <FormattedMessage id="connectionForm.namespaceDefinition.title" />:
+      </Text>
+      <Text size="md" color="grey">
+        <FormattedMessage id={`connectionForm.${namespaceDefinitionOptions[destNamespaceDef]}`} />
+        {namespaceDefinitionOptions[destNamespaceDef] === namespaceDefinitionOptions.customformat && (
+          <>
+            {" - "}
+            {destNamespaceFormat}
+          </>
+        )}
+      </Text>
+    </div>
+  );
+
+  const destinationPrefix = (
+    <div>
+      <Text size="xs" color="grey">
+        <FormattedMessage id="form.prefix" />:
+      </Text>
+      <Text size="md" color="grey">
+        {destPrefix === "" ? (
+          <FormattedMessage id="connectionForm.modal.destinationStreamNames.radioButton.mirror" />
+        ) : (
+          destPrefix
+        )}
+      </Text>
+    </div>
+  );
+
+  const nonBreakingChanges = allowAutoDetectSchema && (
+    <div>
+      <Text size="xs" color="grey">
+        <FormattedMessage id="connectionForm.nonBreakingChangesPreference.label" />:
+      </Text>
+      <Text size="md" color="grey">
+        <FormattedMessage id={`connectionForm.nonBreakingChangesPreference.${nonBreakingChangesPreference}`} />
+      </Text>
+    </div>
+  );
+
+  return (
+    <FlexContainer className={styles.container}>
+      {frequency}
+      {destinationNamespace}
+      {destinationPrefix}
+      {nonBreakingChanges}
```

Comment:
I think this component would be improved if instead of const's, these were components.
```
const NonBreakingChanges = ({ allowAutoDetectSchema, nonBreakingChangesPreference }) => allowAutoDetectSchema ? </> : null;
```
As an idea.

Reply:
Err... I'm not sure I got the idea. What are the advantages?

Reply:
Fixed, all const's were converted to actual components